README.md

```diff
@@ -91,7 +88,6 @@ Example commands:
   cd my-folder
   git clone https://github.com/batoorsohail/leaderboard.git
 ```
--
 
 ### Install
```

Comment:
@batoorsohail You did great job!

- [ ] I like the styling. However, we have some parts in your README.md are missing. It should have instructions about where and how to generate any necessary keys, credentials, how to set, store, and generate them

They are placed in this section:


```
Install
Install this project with:   ----> install with ?

Usage
To run the project, execute the following command: ----> what command ?

Run tests
To run tests, run the following command: ----> what command ?

Deployment
You can deploy this project using: ----> using what ?
```

Kindly fill gaps in this sections. Also you are free to remove unnecessary parts from it such as how to deply.

Reply:
Also, in Acknowledgment:

```
I would like to thank... who/what?
```